### PR TITLE
34 assembler wildcard to quast

### PR DESCRIPTION
**Pull request by DanielSprockett**
Addresses PR #34

### CODE REVIEW

File: resources/snakefiles/assemble.smk [hunk 1]
```diff
@@ -128,8 +129,8 @@ rule metaquast:
                                  assembler=config['assemblers'],
                                  sample=wildcards.sample)
     output:
-        report="output/assemble/metaquast/{sample}/report.html",
-        outdir=directory("output/assemble/metaquast/{sample}")
+        report="output/assemble/{assembler}/metaquast/{sample}/report.html",
+        outdir=directory("output/assemble/{assembler}/metaquast/{sample}")
     threads:
         config['threads']['metaquast']
     log:
```
Comment (tanaes) [inline resources/snakefiles/assemble.smk:136]: Log file also needs `{assembler}`

File: resources/snakefiles/assemble.smk [hunk 2]
```diff
@@ -108,10 +108,11 @@ rule quast:
 
 rule multiqc_assemble:
     input:
-        lambda wildcards: expand("output/assemble/quast/{sample}/report.txt",
+        lambda wildcards: expand("output/assemble/{assembler}/quast/{sample}/report.txt",
+                                 assembler=config['assemblers'],
                                  sample=samples)
     output:
-        "output/assemble/multiqc/multiqc.html"
+        "output/assemble/{assembler}/multiqc/multiqc.html"
```
Comment (tanaes) [inline resources/snakefiles/assemble.smk:115]: There's a conflict here -- the `expand` function in `input` is running multiqc on *all* the assemblers, but the output is per-assembler. We need to pick one or the other.

I'd maybe suggest doing multiqc for the whole shebang, as the point is comparison. So output should be in `output/assemble/multiqc`, and log can remain the same.

File: resources/snakefiles/assemble.smk [hunk 3]
```diff
@@ -128,8 +129,8 @@ rule metaquast:
                                  assembler=config['assemblers'],
```
Comment (tanaes) [inline resources/snakefiles/assemble.smk:129]: for metaquast we to only run on a single assembly, so we don't want to expand `assembler` from `config['assemblers']`; instead, we should pass `assembler=wildcards.assembler` and have `assembler` populated via and `expand` statement in `multiqc_metaquast`

**Review by tanaes — CHANGES_REQUESTED** [review]
A few conflicts found in the assembler wildcards. Now that PR#40 is merged, would also be useful to pull main.

**Comment by DanielSprockett** [comment]
I think I may have made a mistake when merging the previous pull request, but I think I've made all of the requested changes.

**Comment by DanielSprockett** [comment]
wait, no, I think I accidentally pushed these changes to PR #35.

**Comment by DanielSprockett** [comment]
See also:
https://github.com/DanielSprockett/sn-mg-pipeline/commit/7bdaf17aec9ae7deca640771e01f20b14593e0bc

I believe I fixed everything in that branch, as well.

**Pull request merged**